src/java/org/apache/cassandra/db/virtual/SettingsTable.java

```diff
@@ -59,63 +57,104 @@ final class SettingsTable extends AbstractVirtualTable
                            .addPartitionKeyColumn(NAME, UTF8Type.instance)
                            .addRegularColumn(VALUE, UTF8Type.instance)
                            .build());
-        this.config = config;
+        for (String name : BACKWARDS_COMPATABLE_NAMES.keySet())
+        {
+            if (DatabaseDescriptor.hasProperty(name))
+            {
+                throw new AssertionError(String.format("Name '%s' is present in Config, this adds a conflict as this " +
+                                                       "name had a different meaning in the '%s",
+                                                       name, SettingsTable.class.getSimpleName()));
+            }
+        }
+    }
+
+    @Override
+    protected void applyColumnUpdate(AbstractMutableVirtualTable.ColumnValues partitionKey,
+                                     AbstractMutableVirtualTable.ColumnValues clusteringColumns,
+                                     Optional<AbstractMutableVirtualTable.ColumnValue> columnValue)
+    {
+        String name = partitionKey.value(0);
+        String value = columnValue.map(v -> v.value().toString()).orElse(null);
+        if (value == null)
+            throw new InvalidRequestException("Setting the value to null is equivalent to the deletion operation. " +
+                                              "Column deletion is not supported by table vts.settings");
```

Comment:
vts.settings should be probably "system_views.settings" or something like that. Use constants for rendering that string if they are somewhere available.

Reply:
Fixed.